sys/log/full/src/log_fcb.c

```diff
@@ -97,46 +100,185 @@ log_fcb_append(struct log *log, void *buf, int len)
 }
 
 static int
-log_fcb_append_mbuf(struct log *log, struct os_mbuf *om)
+log_fcb_first_chunk_sz(uint8_t align)
 {
+    uint8_t mod;
+
+    mod = sizeof (struct log_entry_hdr) % align;
```

Comment:
% operator is very costly. I think we can assume that the alignment restriction is a power of 2, and then do the math assuming that.